src/shared/components/iframe-runtime.tsx

```diff
@@ -117,9 +121,24 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage]);
+  },[addLocalLinkedDataListener, authoredState, logRequestData, report, setHint, url, initMessage, view]);
 
-  const iframeStyle = iframeStyling ?? {width: "100%", height: iframeHeight, border: "none"};
+  let scaledIframeStyle = undefined;
+  if (scale && report && view !== "standalone") {
+    scaledIframeStyle = {
+      border: "none",
+      height: `${300/scale}px`,
+      transform: `scale(${scale})`,
+      transformOrigin: "left top",
+      width: `${100/scale}%`
+    };
+  }
+
+  const iframeStyle = scaledIframeStyle
+                        ? scaledIframeStyle
+                        : iframeStyling
+                          ? iframeStyling
+                          : {width: "100%", height: iframeHeight, border: "none"};
```

Comment:
I would expect that iframeStyling should override the scale setting. So if a user of the IframeRuntime component sets the iframeStyling this would override the scale based styling.